### PR TITLE
added 2 lines in the VRT to NOT display the NoData values

### PR DESCRIPTION
**Pull request by RousseauLambertLP**

### CODE REVIEW

File: geomet_climate/tileindex.py [hunk 1]
```diff
@@ -201,13 +203,19 @@ def create_dataset(layer_info, input_dir, output_dir):
                                 layer_info['filename'])
 
         LOGGER.info('Generating GPKG')
+        nodata_check = False
         for key in file_time:
             LOGGER.debug('Adding feature to layer')
             if not key.endswith('.tif'):
+                if nodata_check is False:
+                    netcdf_ds = gdal.Open(filename)
```
Comment (tomkralidis) [inline geomet_climate/tileindex.py]: Can you close the file i.e. `netcdf_ds = None`.

File: geomet_climate/tileindex.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 import os
 
 import click
-from osgeo import ogr, osr
+from osgeo import ogr, osr, gdal
```
Comment (tomkralidis) [inline geomet_climate/tileindex.py]: Can we keep the alphabetical order i.e. `from osgeo import gdal, ogr, osr`

File: geomet_climate/tileindex.py (hunk 1 above)
Comment (RousseauLambertLP) [reply geomet_climate/tileindex.py]: done

File: geomet_climate/tileindex.py (hunk 2 above)
Comment (RousseauLambertLP) [reply geomet_climate/tileindex.py]: also done

**Pull request merged**